spec/acceptance/prefix_webhook_spec.rb

```diff
@@ -1,9 +1,9 @@
 require 'spec_helper_acceptance'
 require 'json'
 describe 'Prefix Enabled,System Ruby with No SSL, Not protected, No mcollective' do
-
   context 'default parameters' do
-    let(:pp) {"
+    let(:pp) do
+      "
```

Comment:
That's what rubocop says is best practice - do instead of { and don't start the block on the same line as the do.